bin/gg

```diff
@@ -20,7 +20,12 @@ switch (cli.command) {
 	case 'cl':
 	case 'clone':
 		var repository = cli.args[0];
-		gg.clone(repository);
+
+		if (repository.substring(0, 7) == 'http://') { // URL.
```

Comment:
You should be able to do that currently with `gg cl git@github.com:<username>/<repo>.git`.